lib/expartisan.ex

```diff
@@ -91,6 +106,22 @@ defmodule ExPartisan do
   @doc """
   Sends a message to a process living on `node`. This is analogous to
   `Kernel.send/2`.
+
+  ## Examples
+
+      iex(bar@127.0.0.1)> self
+      #PID<0.308.0>
+
+
+      iex(foo@127.0.0.1)> ExPartisan.send_message(bar, :c.pid(0,308,0), :hello)
+      :ok
+
+      iex(bar@127.0.0.1)4> flush
+      :hello
+      :ok
+
+
+  Note that `server_ref` can be something other than a pid.
```

Comment:
``server_ref`` could be specified here using the type signature from the Erlang documentation

Reply:
The generated hexdocs will have a link in the typespec to its definition, I just pointed it out so people won't assume that's the case based on the given example.